teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    super(name, age, parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_survive
+    true
+  end
```

Comment:
- [ ] Good job on making the Teacher class inherit from the Person class 👍 However I would like to advise you to take notice of the typo you made in the method name, this might cause issues later on when using this method.
- [ ] [OPTIONAL] It is a code style convention to put a question mark behind a method if it returns a boolean value (true or false) or an object to indicate a true value (or “truthy” value).